logic/src/commonMain/kotlin/com/wire/kalium/logic/data/keypackage/KeyPackageRepository.kt

```diff
@@ -37,10 +37,9 @@ class KeyPackageDataSource(
         clientRepository.currentClientId().flatMap { selfClientId ->
             userIds.map { userId ->
                 wrapApiRequest {
-                    keyPackageApi.claimKeyPackages(idMapper.toApiModel(userId))
-                }.flatMap { keyPackageList ->
-                    // TODO: filtering out key packages from the self user client (will be removed soon when BE is updated).
-                    Either.Right(keyPackageList.keyPackages.filter { it.clientID != selfClientId.value })
+                    keyPackageApi.claimKeyPackages(idMapper.toApiModel(userId), selfClientId.value)
+                }.flatMap {
+                    Either.Right(it.keyPackages)
                 }
```

Comment:
Small syntax suggestion  👍🏼
```suggestion
                }.map { it.keyPackages }
```